### PR TITLE
Use token from cookies. Remove usage of koa-redis-session

### PR DESCRIPTION
**Pull request by scherbenokk**

### CODE REVIEW

File: src/client/views/index.html [hunk 1]
```diff
@@ -1,6 +1,20 @@
 <!DOCTYPE html>
 <html class="show-poster">
   <head>
+      <script>
+        window.config = {{{json config}}};
```
Comment (IharKrasnik) [inline src/client/views/index.html]: Think how to move it to react code without blinking rendering

**Comment by scherbenokk** [comment]
@IharKrasnik updated

**Comment by scherbenokk** [comment]
@IharKrasnik updated and checked

File: src/client/helpers/api/api.client.js [hunk 2]
```diff
@@ -17,17 +26,84 @@ const throwApiError = ({ data = {}, status = 500 }) => {
...
+    method: 'post',
+    url: '/account/refresh-token',
+  };
+
+  return api(options);
+};
+
+const isOkStatus = status => status >= 200 && status < 300;
+
+const tryToUpdateToken = async () => {
```
Comment (IharKrasnik) [inline src/client/helpers/api/api.client.js]: Rename to refreshTokenIfNeeded to make it more descriptive

File: src/server/config/routes.js [hunk 3]
```diff
@@ -1,75 +1,30 @@
-const axios = require('axios');
 const Router = require('koa-router');
-const jwt = require('jsonwebtoken');
-const _ = require('lodash');
 
-const config = require('config');
-
-const { logger } = global;
+const {
+  isDev, landingLoginUrl, webSocketUrl, apiUrl,
```
Comment (IharKrasnik) [inline src/server/config/routes.js]: Please don't use destructuring on config importing, import full config and access through it

File: src/client/app.constants.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export const COOKIES = {
+  ACCESS_TOKEN: 'access_token',
+  EXPIRED_AT: 'expired_at',
```
Comment (IharKrasnik) [inline src/client/app.constants.js]: Do we need expire_at at all?

Related comment here: https://github.com/paralect/koa-api-starter/pull/42/files

File: src/client/helpers/api/api.client.js [hunk 5]
```diff
@@ -19,17 +26,79 @@ const throwApiError = ({ data = {}, status = 500 }) => {
...
+  if (!isOkStatus(tokenResponse.status)) {
+    throwApiError({ data: { errors: generalError }, status: tokenResponse.status });
+  }
+};
+
+const redirectToLogin = () => {
+  window.location.href = config.landingLoginUrl;
+};
+
+const handleResponse = async (response, skipUnauthorized) => {
```
Comment (IharKrasnik) [inline src/client/helpers/api/api.client.js]: Please expose `skipUnauthorized` as a named parameter with default = false, as it's not clear from the call, what the purpose of the parameter 

So the call will look like: `await handleResponse(await api(options), { skipUnauthorized: true });`

**Pull request merged**